src/components/TokenBalanceCard/index.tsx

```diff
@@ -1,15 +1,12 @@
 import { FC } from "react"
 import { Token } from "../../enums"
 import TokenBalanceCardTemplate from "./TokenBalanceCardTemplate"
-import KeepCircleBrand from "../../static/icons/KeepCircleBrand"
-import NuCircleBrand from "../../static/icons/NuCircleBrand"
-import T from "../../static/icons/Ttoken"
 import { useToken } from "../../hooks/useToken"
 import { tBTCFillBlack } from "../../static/icons/tBTCFillBlack"
 import { TokenBalanceProps } from "../TokenBalance"
 
 export type TokenBalanceCardProps = {
-  token: Exclude<Token, Token.TBTC>
+  token: Extract<Token, Token.TBTCV2>
```

Comment:
Non-blocking:

Ideally we would not want to use `Extract` here in case we add other tokens to the `Token` anum. I will address that in a separate PR where I remove `useFetchTvl` hook and get rid of other tokens. For now, let's keep it this way.